greedo-layout/src/main/java/com/fivehundredpx/greedolayout/GreedoLayoutSizeCalculator.java

```diff
@@ -76,7 +80,7 @@ public int getFirstChildPositionForRow(int row) {
 
     public int getRowForChildPosition(int position) {
         if (position >= mRowForChildPosition.size()) {
-            computeChildSizesUpToPosition(position);
+            computeChildSizesUpToPosition(position + 1);
```

Comment:
Alternatively we could update the check in `computeChildSizesUpToPosition` so we get the desired behaviour anywhere that method is called.